personalized_active_learning/models/definitions/baseline.py

```diff
@@ -9,7 +9,7 @@ def __init__(
         self,
         output_dim=2,
         text_embedding_dim=768,
-        **kwargs,  # TODO: eliminate kwargs
+        **kwargs,  # TODO: eliminate kwargs / We need it to define different models
```

Comment:
We need to think about an option of defining different types of models with different set of arguments. `Kwargs` can be used as a `garbage params colector`. Not very clean, but quick to use.

Reply:
:heavy_plus_sign: :one: